### PR TITLE
add swagger-ui css/js source maps

### PR DESCRIPTION
**Pull request by WhiteRabbitDE**
| Q             | A
| ------------- | ---
| Bug fix?      |  no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | -
| Fixed tickets | #3008 
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: update-js.sh [hunk 1]
```diff
@@ -8,8 +8,12 @@ if [ -d $dest ]; then
 fi
 mkdir -p $dest
 cp node_modules/swagger-ui-dist/swagger-ui-bundle.js $dest
+cp node_modules/swagger-ui-dist/swagger-ui-bundle.js.map $dest
 cp node_modules/swagger-ui-dist/swagger-ui-standalone-preset.js $dest
+cp node_modules/swagger-ui-dist/swagger-ui-standalone-preset.js.map $dest
 cp node_modules/swagger-ui-dist/swagger-ui.css $dest
+cp node_modules/swagger-ui-dist/swagger-ui.css.map $dest
+
```
Comment (alanpoulain) [inline update-js.sh]: Line to remove.

Comment (WhiteRabbitDE) [reply update-js.sh]: Removed.

**Comment by Lucas-C** [comment]
I'd love to have this !
Thank you @WhiteRabbitDE.
There just seem to be 1 remaining file conflict

**Comment by WhiteRabbitDE** [comment]
> I'd love to have this !
> Thank you @WhiteRabbitDE.
> There just seem to be 1 remaining file conflict

I didn't notice the conflict, my bad ! It's resolved now. :-)

**Comment by Lucas-C** [comment]
Thank you !
@dunglas @teohhanhui : are you ok to merge this ? :)

**Comment by teohhanhui** [comment]
@WhiteRabbitDE You need to actually run this script! :wink: Otherwise the files still wouldn't be there.

**Comment by teohhanhui** [comment]
Anyway, I think we can target `2.4` for this. It's just adding some source map files after all...

**Comment by Lucas-C** [comment]
It's good that `swagger-ui` uses [a recent version of `webpack-sources`](https://github.com/swagger-api/swagger-ui/blob/master/package-lock.json#L22719), because [before `1.4.1` a nasty bug broke webpack-generated sourcemaps](https://github.com/webpack/webpack/issues/8302#issuecomment-516964714)

**Comment by teohhanhui** [comment]
@WhiteRabbitDE would you like to take care of changing the branch?

**Comment by WhiteRabbitDE** [comment]
> @WhiteRabbitDE would you like to take care of changing the branch?

I forgot to git add the missing files, they are added now. Maybe a stupid question but can i change the target branch without pushing all other commits since i branched the core ?

**Comment by teohhanhui** [comment]
In order to switch the target branch from `master` to `2.4`, you should use `git cherry-pick` :smile:

If you'd like to give it a try, go ahead.

**Comment by teohhanhui** [comment]
Just let me know if you'd like me to handle this. :smile:

**Comment by WhiteRabbitDE** [comment]
> Just let me know if you'd like me to handle this. 😄

I'm feeling dumb right now, sorry. May be it's too friday for me 😄 I would appreciate your help!

**Comment by teohhanhui** [comment]
Done.

**Comment by teohhanhui** [comment]
Thanks @WhiteRabbitDE! :tada:

**Comment by Lucas-C** [comment]
Yes, thanks ! 👍

**Pull request merged**